styles/normalize.css

```diff
@@ -347,3 +347,23 @@ template {
 [hidden] {
     display: none;
 }
+
```

Comment:
These styles seem misplaced in this global file. Maybe putting them in a styled-components wrapper around the carousel would be neat?

Reply:
@nutzaalex do you think that'd be the right approach?

Reply:
yeah, so the normalize.css should only deal with css site wide normalisation for all the different browsers inconsistencies, I agree that they should be moved to the component which is using them and converted to styled